Integrate vault dashboard to image

francoisferrand opened the pull request:
Issue: ZENKO-3776

francoisferrand committed:
Integrate vault dashboard to image

Issue: ZENKO-3776

francoisferrand committed:
Deploy Cloudserver dashboard

The dashboard must be added to ISO (already done), but also referenced
in zenkoversion CR.

Issue: ZENKO-3747

bert-e commented:
# Hello francoisferrand,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/zenko/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-3776 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.2.4`


Please check the `Fix Version/s` of ZENKO-3776, or the target
branch of this pull request.

francoisferrand committed:
Deploy Vault dashboards

Issue: ZENKO-3776

francoisferrand committed:
Bump vault to 8.3.11

Issue: ZENKO-3776

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

francoisferrand committed:
Fix dashboard images used in e2e test

Issue: ZENKO-3776

francoisferrand committed:
Remove dashboard from e2e deployment

Zenko-operator is not able to access a "public" registry at this time:
it uses http-only, without auth.

Proper e2e testing would require either to add this capability (and let the
operator get the registry credentials), or run e2e tests with a local
registry (static-container-registry or docker registry v2).

Issue: ZENKO-3776

francoisferrand commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.2`



The following branches will **NOT** be impacted:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.1`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.2`



The following branches have **NOT** changed:

* `development/0.9`
* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.1`



Please check the status of the associated issue ZENKO-3776.

Goodbye francoisferrand.
